beacon-chain/sync/rpc_status.go

```diff
@@ -208,12 +209,32 @@ func (r *Service) validateStatusMessage(msg *pb.Status, stream network.Stream) e
 		return errWrongForkDigestVersion
 	}
 	genesis := r.chain.GenesisTime()
+	finalizedEpoch := r.chain.FinalizedCheckpt().Epoch
 	maxEpoch := slotutil.EpochsSinceGenesis(genesis)
 	// It would take a minimum of 2 epochs to finalize a
 	// previous epoch
 	maxFinalizedEpoch := maxEpoch - 2
```

Comment:
Kinda wish 2 is not a magic number, but I understand the lower bound to finalize an epoch is not explicitly defined